activejdbc/src/main/java/org/javalite/activejdbc/Registry.java

```diff
@@ -529,4 +543,11 @@ protected String getTableName(Class<? extends Model> modelClass) {
     private void registerColumnMetadata(String table, Map<String, ColumnMetadata> metaParams) {
         metaModels.setColumnMetadata(table, metaParams);
     }
-}
+
+    public String getModelFile() {
+        return modelFile;
+    }
+
+    public void setModelFile(String modelFile) {
```

Comment:
Here is the problem: since  you are opening the `setModelFile` to the outside world,  you will have the following issue: 

* Anyone can call it any time. If it is called before registration, you are good. If it was called after registration, this has no effect. 

Not sure about the best way to implement this, but the implementation needs a more careful approach.

Reply:
Yes, it is intended that the method be required to be called before registration.

Reply:
I understand it, what I'm trying to say, this maybe clear for you but will be confusing to anyone else.

Reply:
Sure, I can close this, I was just trying to solve my issue since I didn't notice any activity towards a resolution.

Reply:
I can accept this though. Maybe you have an idea to solve this in a bit more elegant way?

Reply:
The elegant method I thought about was to add it into activejdbc.properties, unfortunately due to the root cause of the issue that this PR addresses, the same issue would persist with activejdbc.properties.

Reply:
haha, I thought the same. Please add extensive JavaDoc to the `setModelFile` method, and I will merge

Reply:
Updated, I think I made it very clear in the javadocs to only use the method if you're 100% sure you know how it works.

Reply:
JavaDoc is looking  good, thanks